reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -11110,6 +11138,14 @@ static <I> Flux<I> wrap(Publisher<? extends I> source) {
 		return target;
 	}
 
+	static final Consumer<? super AutoCloseable> AUTO_CLOSE = resource -> {
+		try {
+			resource.close();
+		} catch (Exception e) {
+			throw new RuntimeException(e);
```

Comment:
I wonder whether `Exceptions.propagate(t)` would be a good candidate here. It feels like a natural choice.

Reply:
✅ updated, `Exceptions.propagate(t)` is what I'm looking for!